IntegrationTests/Services/AWSS3IntegrationTests/S3XCTestCase.swift

```diff
@@ -116,4 +116,12 @@ class S3XCTestCase: XCTestCase {
         let input = DeleteBucketInput(bucket: bucketName)
         _ = try await client.deleteBucket(input: input)
     }
+
+    static func generateRandomTextData(ofSizeInMB megabytes: Double) -> Data {
```

Comment:
This method should take a precise integer number of bytes.

I'm okay with it if we have a convenience form that takes a Double number of MB then converts that to bytes.

Also, as mentioned above, let's move this to integration test utils.